.travis.yml

```diff
@@ -4,7 +4,8 @@ python:
   - "2.7"
   - "3.2"
   - "3.3"
+  - "3.4"
 # command to install dependencies
-install: "pip install -r requirements.txt --use-mirrors"
```

Comment:
`use-mirrors` is depreciated (http://pip.readthedocs.org/en/stable/news/ - see release 7.0.0)